Source/PopupView.swift

```diff
@@ -492,8 +507,18 @@ public struct Popup<PopupContent: View>: ViewModifier {
 
     /// This is the builder for the sheet content
     func sheet() -> some View {
+        let scrollView =
+        ScrollView {
+            view()
+        }
+        .introspect(.scrollView, on: .iOS(.v15, .v16, .v17), customize: { scrollView in
+            configure(scrollView: scrollView)
+        })
+
+        let contentView = type != .scroll ? AnyView(view()) : AnyView(scrollView)
```

Comment:
Please remove AnyView - this leads to many unpredictable bugs, use a @ViewBuilder or some other trick